app/src/main/java/org/fossasia/openevent/general/auth/AuthFragment.kt

```diff
@@ -48,8 +48,10 @@ class AuthFragment : Fragment(), ComplexBackPressFragment {
     private val safeArgs: AuthFragmentArgs by navArgs()
     private val smartAuthViewModel by sharedViewModel<SmartAuthViewModel>()
 
-    override fun onCreate(savedInstanceState: Bundle?) {
-        super.onCreate(savedInstanceState)
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [LongMethod - 61/60 - [onCreateView] at /src/app/src/main/java/org/fossasia/openevent/general/auth/AuthFragment.kt:55:5](https://app.codacy.com/app/fossasia/open-event-android-general/pullRequest?prid=4650690)

Reply:
How can I change the pre-defined function of a fragment?